collection-scripts/gather_ceph_logs

```diff
@@ -67,7 +67,7 @@ CMDS
         mkdir -p "${CSI_LOG_OUTPUT_DIR}"
         mkdir -p "${JOURNAL_OUTPUT_DIR}"
         mkdir -p "${KERNEL_OUTPUT_DIR}"
-        mkdir -p "${COREDUMP_OUTPUT_DIR}"
+        mkdir -p "${CRASH_OUTPUT_DIR}"
```

Comment:
This change is not a part of this PR. Please rebase on main.

Reply:
it is rebased on main only, just got merged with the latest PR merged of yours, hence showing this.
manually reverting it will merge older change in main.
Or will have to create new PR, so I think it should be fine merging this

Reply:
```
- git reset HEAD^1
- Add all the files again except gather_ceph_logs
- Force push
```